File: sortingandsearching/ss1.py
aa=input()
a=[int(x) for x in input().split()]
a.sort()
temp=0
a.append(a[len(a)-1])
for i in range(len(a)-1):
	if (a[i]!=a[i+1]):
		temp=temp+1
print(temp+1)

File: sortingandsearching/ss1.cpp
#include<bits/stdc++.h>
using namespace std;

int main()
{
int a,temp;
cin>>a; 
vector <int> aa; 
for(int i=0; i<a; i++){
cin>>temp;
aa.push_back(temp);
}
sort (aa.begin(),aa.end());
int a2=0;
for (int i=0;i<a;i++){
if (aa[i]!=aa[i-1]) a2=a2+1;
}
cout<<a2; 
return 0;
}
